.envrc

```diff
@@ -3,6 +3,7 @@ export NETWORK=regtest
 export NODE_ENV=development
 
 export LND1_TLS=$(cat dev/lnd/tls.cert.base64)
+export LND1_TLS_KEY=$(base64 dev/lnd/tls.key | tr -d '\n\r')
```

Comment:
why is this needed?

Reply:
note: seems below the value is already converted in `.base64` in the script so a simple `cat` is enough